lib/travis/shell/generator/bash/cmd.rb

```diff
@@ -34,6 +34,7 @@ def opts(options)
               opts << "--display #{escape(options[:echo])}" if options[:echo].is_a?(String)
               opts << '--retry'  if options[:retry]
               opts << '--timing' if options[:timing]
+              opts << "--stage #{options[:stage].to_s}" if !options[:stage].to_s.empty?
```

Comment:
I note that this _*must*_ precede handling of `secure`; `--secure` in `travis_cmd` adds ` 2>/dev/null`, so that this option must be the last one passed to `travis_cmd`.

Reply:
Or, at least, that's what I observed while testing. (I'm now trying to reason about this with bash, but not exactly clear.)